index.js

```diff
@@ -18,8 +18,19 @@ const assertPatternsInput = patterns => {
 	}
 };
 
-const checkCwdOption = options => {
-	if (options && options.cwd && !fs.statSync(options.cwd).isDirectory()) {
+const checkCwdOption = (options = {}) => {
+	if (!options.cwd) {
+		return;
+	}
+
+	let stat;
+	try {
+		stat = fs.statSync(options.cwd);
```

Comment:
Do we want to do this synchronously?

Reply:
We probably want a separate async version. I might have been too lazy to do that initially or just forgot. PR welcome :)

Reply:
Yeah, I think we'll need it since we use `globby` in a lot of modules and this might result in a decrease in performance.